### DISCUSSION TITLE
Where is the code for "Unable to Load Video"?

### DISCUSSION THREAD

**Discussion by davegoopot** [mozilla/hubs#2989] [General Questions & Discussion] [answered]
I've put a link element in my Hub with a URL that doesn't link to an image or model - just a regular web page. When viewing this it is rendered with a message saying "Unable to Load Video". I was curious as to what code was running to show this message. I've searched for the word "Unable" in this repo [1] but I can't find it. Could someone point me to where this message comes from in the code please?

[1] https://github.com/mozilla/hubs/search?q=Unable&unscoped_q=Unable

**Comment by misslivirose** [comment, marked as answer]
I believe that this is handled by reticulum in media_resolver.ex: https://github.com/mozilla/reticulum/blob/master/lib/ret/media_resolver.ex

**Reply by davegoopot** [reply]
If that's the case, shouldn't I be able to find the word "unable" somewhere in the source? As in "Unable to Load Video".

**Reply by misslivirose** [reply]
The error messages that are loaded when media doesn't resolve are png images that display instead of the requested content, so it wouldn't show up in a text search.

**Reply by emclaren** [reply]
For reference - it can be found here: https://github.com/mozilla/reticulum/blob/master/assets/static/quota-error.png
